app/portainer/rest/webhook.js

```diff
@@ -0,0 +1,8 @@
+angular.module('portainer.app')
```

Comment:
The remove method should be added in `rest/webhooks.js` and this file can be removed.